packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -103,180 +86,121 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
+                  <NavLink.Tooltip label={formatMessage(link.intlLabel)}>
+                    <NavLink.Icon>
+                      <LinkIcon fill="neutral500" />
+                    </NavLink.Icon>
+                    {badgeContent && (
+                      <NavLink.Badge label="locked" background="transparent" textColor="neutral500">
+                        {badgeContent}
+                      </NavLink.Badge>
+                    )}
+                  </NavLink.Tooltip>
+                </NavLink.Link>
               );
-            })}
-          </NavSection>
-        ) : null}
-
-        {generalSectionLinks.length > 0 ? (
-          <NavSection
-            label={formatMessage({
-              id: 'app.components.LeftMenu.general',
-              defaultMessage: 'General',
-            })}
-          >
-            {generalSectionLinks.map((link) => {
+            })
+          : null}
+        {generalSectionLinks.length > 0
+          ? generalSectionLinks.map((link) => {
               const LinkIcon = link.icon;
 
+              const badgeContent =
+                link.notificationsCount && link.notificationsCount > 0
+                  ? link.notificationsCount.toString()
+                  : undefined;
+
               return (
-                <NavLink
-                  as={RouterNavLink}
-                  badgeContent={
-                    link.notificationsCount && link.notificationsCount > 0
-                      ? link.notificationsCount.toString()
-                      : undefined
-                  }
-                  // @ts-expect-error the props from the passed as prop are not inferred // joined together
-                  to={link.to}
-                  key={link.to}
-                  icon={<LinkIcon />}
-                  onClick={() => handleClickOnLink(link.to)}
-                >
-                  {formatMessage(link.intlLabel)}
-                </NavLink>
+                <NavLink.Link to={link.to} key={link.to} onClick={() => handleClickOnLink(link.to)}>
+                  <NavLink.Tooltip label={formatMessage(link.intlLabel)}>
+                    <NavLink.Icon>
+                      <LinkIcon fill="neutral500" />
+                    </NavLink.Icon>
+                    {badgeContent && (
+                      <NewNavLinkBadge label={badgeContent} backgroundColor="primary600">
+                        {badgeContent}
+                      </NewNavLinkBadge>
+                    )}
+                  </NavLink.Tooltip>
+                </NavLink.Link>
               );
-            })}
-          </NavSection>
-        ) : null}
-      </NavSections>
-
-      <NavFooter>
-        <NavUser
-          id="main-nav-user-button"
-          ref={navUserRef}
-          onClick={handleToggleUserLinks}
-          initials={initials}
+            })
+          : null}
+      </NavListWrapper>
+      <NavUser
```

Comment:
Why wouldnt we have the menu & the User section all in the same component? Is there a benefit to the menu live outside the declaration for it's trigger?

Reply:
Also reviewing the semantics this should be a `menu`. I'd look at using the `Menu` component instead, this will handle stuff we expect from a popover. It probably means we can remove the `onBlur` handler, which im not totally clear the point of.

Reply:
ok I refactored the component, now all the logic of the NavUser is inside the component